docs/csharp/language-reference/tokens/interpolated.md

```diff
@@ -22,38 +22,42 @@ ms.author: "ronpet"
 
 The `$` special character identifies a string literal as an *interpolated string*. An interpolated string looks like a template string that contains *interpolated expressions*. When the interpolated string is resolved, for example in an assignment statement or a method call, interpolated expressions are replaced by the string representations of their results to produce the result string. This feature is available in C# 6 and later versions.
 
-String interpolation is a more convenient way to create formatted strings than a [string composite formatting](../../../standard/base-types/composite-formatting.md). The following example uses both features to produce the same output:
+String interpolation is a more convenient way to create formatted strings than a [string composite formatting](../../../standard/base-types/composite-formatting.md) feature. The following example uses both features to produce the same output:
 
 [!code-csharp-interactive[compare with composite formatting](../../../../samples/snippets/csharp/language-reference/tokens/string-interpolation.cs#1)]
 
-> [!IMPORTANT]
+> [!NOTE]
 > You cannot have any white space between the `$` and the `"` that starts the string. Doing so causes a compile-time error.
 
 The structure of an item with an interpolated expression is as follows:
 
 ```
-{<interpolated-expression>[,<alignment>][:<format-string>]}
+{<interpolated-expression>[,<alignment>][:<formatString>]}
```

Comment:
Why is `interpolated-expression` separating words using a hyphen, but `formatString` is using camelCase? I think both should use the same approach, whichever one that is.

Reply:
I've switched to `formatString` to be consistent with [composite formatting doc](https://docs.microsoft.com/en-us/dotnet/standard/base-types/composite-formatting#format-item-syntax), so everything then should be in camelCase. Thank you for the catch @svick